### ISSUE TITLE
Does POP3 NTLM authentication mechanisms works correctly?

### ISSUE THREAD

**Issue by SuhovDE** [jstedfast/MailKit#391]
Hello,

I tried to get email using Pop3Client, but received the following exception:

> MailKit.Net.Pop3.Pop3CommandException POP3 server did not respond with a +OK response to the STAT command.  

POP3 log contains the following lines:
```
Connected to pop://xxx.xxx.xxx.xxx:110/
S: +OK Microsoft Exchange Server 2003 POP3 server version 6.5.7638.1 (domain.net) ready.
C: CAPA
S: +OK Capability list follows
S: TOP
S: USER
S: PIPELINING
S: EXPIRE NEVER
S: UIDL
S: SASL NTLM
S: .
C: AUTH NTLM
S: +OK
C: CAPA
S: -ERR Logon failure: unknown user name or bad password.
C: STAT
S: -ERR Command is not valid in this state.
C: QUIT
S: +OK Microsoft Exchange Server 2003 POP3 server version 6.5.7638.1 signing off.
```
Is it correct that client send CAPA as answer for +OK? I thought, that it should send Base 64 encoded Type 1 message instead. See [NTLM POP3 Authentication](https://curl.haxx.se/rfc/ntlm.html#ntlmPop3Authentication)

FYI: After I cleared AuthenticationMechanisms property, the client was authenticated correctly, but using USER Mechanisms.

**Comment by jstedfast** [comment]
Okay, so the problem is that the POP3 server should not be sending `+OK` (which means "command finished successfully"), it should be sending just `+` (which means "go ahead and continue").

**Comment by jstedfast** [comment]
I've committed a patch that attempts to work around this broken server behavior. Can you give it a try and report back as to whether or not it fixes the issue?

**Comment by SuhovDE** [comment]
With this patch it works correctly.

Thank you for fix.

**Comment by jstedfast** [comment]
You're welcome!

**Comment by jstedfast** [comment]
I've released MailKit 1.6.0 with this fix
